shell/aliases/rails.sh

```diff
@@ -122,6 +122,9 @@ function psql
   if [[ "$(__database_yml adapter)" == 'postgresql' ]]; then
     PGDATABASE="$(__database_yml database)" command psql "$@"
     return $?
+  elif [[ "$(__database_yml url)" =~ '^postgres' ]]; then
```

Comment:
In my experience, at least on bash 3.x which comes with macOS and bash 5.x from Homebrew, the right hand side of `=~` is taken literally and should not be quoted. Is this working for you? What shell are you on?

```console
$ [[ "$(echo postgresql://foo)" =~ '^postgres' ]]; echo $?
1
$ [[ "$(echo postgresql://foo)" =~ ^postgres ]]; echo $?
0
```

Reply:
```
$ [[ "$(echo postgresql://foo)" =~ '^postgres' ]]; echo $?
0
$ [[ "$(echo postgresql://foo)" =~ ^postgres ]]; echo $?
0
$ echo "$SHELL"
/bin/zsh
```

Reply:
Ah, of course: it’s another difference between bash and zsh.